<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SHAPESHIFT KEEPKEY AUTHENTICATOR</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/icon.png</normaloff>../icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>46</y>
      <width>450</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
selection-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Initialize with QR Code or generate OTP on Keepkey</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qrscreencap">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>180</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
QPushButton::hover
{
background-color : lightgreen;
}</string>
    </property>
    <property name="text">
     <string>Read QR Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otpgen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>180</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
QPushButton::hover
{
background-color : lightgreen;
}</string>
    </property>
    <property name="text">
     <string>Generate OTP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>180</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
QPushButton::hover
{
background-color : lightgreen;
}</string>
    </property>
    <property name="text">
     <string>Connect KeepKey</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>180</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
QPushButton::hover
{
background-color : lightgreen;
}</string>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionAbout"/>
    <addaction name="actionUpdates"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQuit_Program">
   <property name="text">
    <string>Quit Program</string>
   </property>
  </action>
  <action name="actionQuit_FromProgram">
   <property name="text">
    <string>Quit Program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit Application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdates">
   <property name="text">
    <string>Check For Updates</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
